app/views/support_interface/tasks/index.html.erb

```diff
@@ -56,7 +56,7 @@
   <section class="app-section app-section--with-top-border">
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
-        <h2 class="govuk-heading-m">Generate test applications</h2>
+        <h2 class="govuk-heading-m">Generate test applications for the <%= RecruitmentCycle.current_year %> recruitment cycle</h2>
```

Comment:
does this need to be `real_current_year?`

Reply:
Yes, I thought I had put that in, that was the reason I made that method :grimacing: .

Reply:
Oh sorry, after looking at it again, that should be `current_year`.